src/ESLintLinter.php

```diff
@@ -63,6 +63,8 @@ protected function getMandatoryFlags() {
     return array(
       '--format=json',
       '--no-color',
+      '--resolve-plugins-relative-to',
+      parent::getNodeCwd(),
```

Comment:
`parent::` should really only be used when you want to explicitly ignore the definition (if any) of this method within the current class (most often when wanting to call the parent method from this class's method of the same name).

That doesn't seem to be the case here, so this should just use `$this->getNodeCwd()`